tests/test_main.py

```diff
@@ -577,6 +578,22 @@ class Config:
     assert m.foo == 'foo'
 
 
+@pytest.mark.skipif(not Literal, reason='typing_extensions not installed')
+def test_literal_enum_values():
+    FooEnum = Enum('FooEnum', {'foo': 'foo', 'bar': 'bar'})
```

Comment:
```suggestion
    FooEnum = Enum('FooEnum', {'foo': 'foo_value', 'bar': 'bar_value'})
```

Makes the test clearer, otherwise we could be using `FooEnum.foo.name`